csharp/src/Drivers/Apache/Spark/README.md

```diff
@@ -40,7 +40,7 @@ but can also be passed in the call to `AdbcDatabase.Connect`.
 | `adbc.apache.statement.batch_size` | Sets the maximum number of rows to retrieve in a single batch request. | `50000` |
 | `adbc.apache.statement.polltime_ms` | If polling is necessary to get a result, this option sets the length of time (in milliseconds) to wait between polls. | `500` |
 | `adbc.apache.statement.query_timeout_s` | Sets the maximum time (in seconds) for a query to complete. Values can be 0 (infinite) or greater than zero. | `60` |
-| `adbc.http_options.tls.enabled` | If tls needs to enabled or not. One of `True`, `False` | `False` |
+| `adbc.http_options.tls.enabled` | If tls needs to enabled or not. One of `True`, `False` | `True` for `adbc.spark.type` with value `databricks`, `False`, otherwise. |
```

Comment:
I would argue that the real problem with the previous change is that in 2025 no one should disable TLS by default.

Reply:
> I would argue that the real problem with the previous change is that in 2025 no one should disable TLS by default. We should instead default this to `True` (except, perhaps, when using port 80) and force people into the insecure setting.

So, instead, the caller must ensure to set the `adbc.http_options.tls.enabled` property to `true` for managed services?

I'm okay with that. I can remove the "default" behavior.

However, I'd like add the test configuration changes so that it can be set with a test configuration - that was missing in the PR that introduced those options.

Reply:
> I would argue that the real problem with the previous change is that in 2025 no one should disable TLS by default. We should instead default this to `True` (except, perhaps, when using port 80) and force people into the insecure setting.

Ah. Okay. That's much BETTER. (Ignore previous comment - did not read carefully.)

Reply:
TLS enabled set to `True` is now the default for all HTTP-based drivers.